src/Http/Http.Extensions/src/HttpJsonServiceExtensions.cs

```diff
@@ -0,0 +1,32 @@
+// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+
+using Microsoft.AspNetCore.Http.Json;
+
+namespace Microsoft.Extensions.DependencyInjection;
+
+/// <summary>
+/// Extension methods to configure JSON serialization behavior.
+/// </summary>
+public static class RouteHandlerJsonServiceExtensions
+{
+#pragma warning disable CS0419 // Ambiguous reference in cref attribute
+    /// <summary>
+    /// Configures options used for reading and writing JSON by route handlers.
+    /// </summary>
+    /// <remarks>
+    /// The options configured here will only affect JSON returned and processed
+    /// from route handlers, not controllers, when using <see cref="System.Net.Http.Json.HttpContentJsonExtensions.ReadFromJsonAsync" />
```

Comment:
```suggestion
    /// from route handlers, not controllers, when using <see cref="Microsoft.AspNetCore.Http.HttpRequestJsonExtensions.ReadFromJsonAsync" />
```

Can we remove `#pragma warning disable CS0419` after this change?

Reply:
Oh snap! Good catch.

> Can we remove #pragma warning disable CS0419 after this change?

Unfortunately not. It's complaining about the ambiguity between the different overloads we have for `ReadFromJsonAsync` and `WriteAsJsonAsync` methods.

Reply:
Maybe we should just pick the most common overload so it doesn't break the links generated by the tooling.

```
<see cref="HttpRequestJsonExtensions.ReadFromJsonAsync{TValue}(HttpRequest, CancellationToken)" />
```

Reply:
Does this work: https://stackoverflow.com/a/18193169?

Reply:
Also see https://github.com/dotnet/csharplang/issues/320

Reply:
I saw https://github.com/dotnet/csharplang/issues/320 but that was suggesting `O:` instead of `M:`. I think the safest thing is just to pick an overload.

Reply:
Decided to go with `O:` since it does the job well and is [used wisely elsewhere in the dotnet repos](https://dotnet.grep.app/search?q=%22O%3A).